app/views/unattended/snippets/_pxegrub2_chainload.erb

```diff
@@ -0,0 +1,10 @@
+<%#
+kind: snippet
+name: pxegrub2_chainload
+%>
+menuentry "Chainload into bootloader on first disk" {
+  insmod part_msdos
+  insmod chain
+  chainloader (hd0,0)+1
```

Comment:
I just retested this. Still does not work.

![image](https://cloud.githubusercontent.com/assets/4107560/17548062/6eae0b9c-5eea-11e6-9a0f-51a9ac3288b1.png)